content/api/index.md

```diff
@@ -50,3 +50,23 @@ This API reference is generated from our OpenAPI specification. Found an error?
   ---
   ::
 ::
+
+## Advanced Requests
```

Comment:
Could we call this header "Relational Data" or something similar so it will pop up in search if you're looking for "relation" or "data"?

Reply:
Done! Thanks for the suggestion. How does it look, @br41nslug